.github/workflows/integration-test.yaml

```diff
@@ -4,6 +4,7 @@ name: Integration Tests
 
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
LGTM.

If you want save some tBTC, you might consider [running a pull_request_target workflow when a pull request merges](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request_target-workflow-when-a-pull-request-merges).

Reply:
> LGTM.
> 
> If you want save some tBTC, you might consider [running a pull_request_target workflow when a pull request merges](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request_target-workflow-when-a-pull-request-merges).

We have enough tBTC available, so we can conduct tests early.